packages/js/src/ui/components/InboxTabs/InboxTab.tsx

```diff
@@ -26,7 +26,7 @@ export const InboxTabUnreadNotificationsCount = (props: { count: number }) => {
 export const InboxTab = (props: Tab & { class?: ClassName }) => {
   const { status } = useInboxContext();
   const style = useStyle();
-  const unreadCount = useUnreadCount({ filter: { tags: props.value } });
+  const unreadCount = useUnreadCount({ filter: { tags: getTagsFromTab(props) } });
```

Comment:
Because we have deprecated the `value` property on the tab object and introduced the `filter` property instead, I created this helper function that takes the tags value from the new prop or fallbacks to the old. This function is used around the code.

Reply:
Please add this as a comment in the code as in a future major or minor version we can remove the code and remove the deprecation.